Utils/data_analys.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import pandas as pd
+import spacy
+import re
+from collections import Counter
+from nltk import ngrams
+from nltk.tokenize import TweetTokenizer
+
+
+DATA_DIR = '../../Data/Russian Twitter Corpus/'
+
+
+def read_csv_clear(name):
+    '''
+    Читает данные из файла svc, и преобразует их для дальнейшей обработки
+    :param name: имя файла
+    :return: dataframe с подготовленными данными
+    '''
+    x = pd.read_csv(f"{DATA_DIR}" + name, sep=';', header=None, engine='c')
+    tknzr = TweetTokenizer(strip_handles=True, reduce_len=True)
+    for i in range(12):
+        if i != 3:
+            del x[i]
+    for i,_ in enumerate(x[3]):
```

Comment:
Any enumeration of variables should be separated with one space between each variable :)